packages/@snowpack/app-scripts-preact/snowpack.config.js

```diff
@@ -1,27 +1,15 @@
-const fs = require("fs");
-const path = require("path");
+const fs = require('fs');
+const path = require('path');
 
 const cwd = process.cwd();
-const isTS = fs.existsSync(path.join(cwd, "tsconfig.json"));
-
-const scripts = {
-  "mount:public": "mount public --to /",
-  "mount:src": "mount src --to /_dist_",
-};
-
-if (isTS) {
-  scripts["run:tsc"] = "tsc --noEmit";
-  scripts["run:tsc::watch"] = "$1 --watch";
-}
+const isTS = fs.existsSync(path.join(cwd, 'tsconfig.json'));
 
 module.exports = {
-  scripts,
-  plugins: [
-    "@snowpack/plugin-babel",
-    "@prefresh/snowpack",
-    "@snowpack/plugin-dotenv",
-  ],
-  devOptions: {},
+  mount: {
+    public: '/',
+    src: '/_dist_',
+  },
+  plugins: ['@snowpack/plugin-babel', '@prefresh/snowpack', '@snowpack/plugin-dotenv'],
```

Comment:
I know changes like this are noisy, but it was better to upgrade the plugins + template configs in one pass. Now we remove `scripts: {}` from the templates wherever possible.